### ISSUE TITLE
Double Email Headers..??

### ISSUE THREAD

**Issue by angelleye** [angelleye/offers-for-woocommerce#227]
I've seen a couple of different report this happening.

![image](https://cloud.githubusercontent.com/assets/629241/12612711/5dbf3812-c4ba-11e5-8b45-f2f26f92719a.png)

This particular example was reported here:  https://www.angelleye.com/support/scp/tickets.php?id=380

Here is another one:  https://wordpress.org/support/topic/email-issue-6?replies=2#post-8276680

**Comment by freddy-barkhuizen** [comment]
Good day,

When will this issue be resolved? I am also experiencing the double header issue.

Regards,
Freddy

**Comment by angelleye** [comment]
We have been focusing on this plugin recently and will be getting this 1.3.0 update released as soon as we can.  We're hoping to have this fix included in that update.

So hopefully within the next 2 or 3 weeks here at the most.

**Comment by freddy-barkhuizen** [comment]
That is great news, thank you very much.

**Comment by kcppdevelopers** [comment]
I have checked multiple time but for me it's working fine. it seem may be conflict with other woo plugin.

**Comment by angelleye** [comment]
@freddy-barkhuizen Can you please provide details about the plugins you have installed?  We'll need to see if we can reproduce the problem by installing the same plugins.

Or, if you can disable other plugins and run a test offer to see if the issue goes away that would be great.  Then you can re-enable each one and keep trying until the problem comes back to find the culprit.

**Comment by angelleye** [comment]
This user reports it's happening when the WooCommerce PDF Invoices & Packing Slips plugin is installed:  https://wordpress.org/support/topic/2-issue-new-offer-with-2-header-footer-pdf-invoice-creation-error?replies=5#post-8683695

**Comment by keyurptl** [comment]
WooCommerce PDF Invoices & Packing Slips plugin will call WC_Email on every Ajax call so duplication of "woocommerce_email_header"<Action> 
so i removed one of them from action when ajax called.

**Comment by chrwald** [comment]
Hi guys, decembers fix can cause that no headers and footers at all appear on any email when using PDF Invoice and Packing Slips.  Removing the fix solves the problem and causes no double headers and footers - at least in my environment.

**Comment by angelleye** [comment]
@chrwald, so you're saying that our update completely eliminates the headers from your emails?  But then reverting back to the previous version of the plugin only displays a single email header?

Please verify whether or not I'm understanding that correctly.  Thanks!

**Comment by chrwald** [comment]
@angelleye Yes, thats what I encountered. I'm using your plugin in the current version and also the PDF Invoices and Packing Slips Plugin in Version 1.5.38. Both plugins activated the headers and footers from all emails are gone - except, interesstingly, the emails form the offers plugin (Offer received, Counter Offer, etc.), which still have header/footer.

What I tried so far:

- Turning off one of both plugins: Headers/Footers are back again.

- Removing [this fix](https://github.com/angelleye/offers-for-woocommerce/commit/bd224e8298fcbddff1ac68560d852b36e95718f4) regarding this problem, file: public/class-offers-for-woocommerce.php, lines 178-181: Headers/Footers are back - and also not duplicated like in the orginial issue here.
